docs/state.md

```diff
@@ -169,6 +169,27 @@ AppStore
     locale: string, // en_US, en, or any other locale string
     ignoredWords: Array<string>, // List of words to ignore
     addedWords: Array<string> // List of words to add to the dictionary
+  },
+  autofill: {
+    addresses: [{
+      name: string,
+      organization: string,
+      streetAddress: string,
+      city: string,
+      state: string,
+      postalCode: string,
+      country: string,
+      phone: string,
+      email: string,
+      guid: Object.<string, <string>> // map of (partition, id) used to access the autofill entry in database
+    }],
+    creditCards: [{
```

Comment:
we can't store plain text CC data in the state. It's already encrypted on the electron side so we should either store the guid and read from there or encrypt (base64 encoded) here. Preferably the former I think because we don't really need two copies.